.buildkite/scripts/install-kubectl.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo "--- Install kubectl"
+
+MSG="parameter missing."
+DEFAULT_HOME="/usr/local"
+K8S_VERSION=${K8S_VERSION:?$MSG}
+HOME=${HOME:?$DEFAULT_HOME}
+KUBECTL_CMD="${HOME}/bin/kubectl"
+
+if command -v kubectl
```

Comment:
similar to my previous comment, this works because we happened to export PATH in the previous script. I think this is some implicit behavior which can lead to trouble in the future, that's why I think the update of the PATH should be moved somewhere more visible ex, the pipeline